src/bin/ion/commands/beta/generate/templates/rust/struct.templ

```diff
@@ -29,106 +31,52 @@ pub mod  {{ target_kind_name | snake }} {
 
         pub fn read_from(reader: &mut Reader) -> SerdeResult<Self> {
             let mut abstract_data_type = {{ target_kind_name }}::default();
-            {% if abstract_data_type == "Value"%}
```

Comment:
(🗺️ PR tour) This template used to contain all abstract data types templates but now it separated into 3 templates. 
- struct.templ (For struct type)
- scalar.templ
- sequence.templ